react-frontend/src/components/Guides/CollaborationTools/collaborationTools.js

```diff
@@ -1,17 +1,55 @@
 import React from 'react';
-
+import styled from 'styled-components';
+import { createGlobalStyle } from 'styled-components';
 import CollaborationTool from './collaborationTool';
-// import ResourceHeader from './resourceheader';
-// import '@bcgov/bc-sans/css/BCSans.css'
-import '../../../css/collaborationTools.css';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
I think there should be a seperation of what is a global style, a component level style, and a common or reusable style.